lego-webapp/pages/achievements/+Page.tsx

```diff
@@ -91,40 +91,46 @@ const AchievementGroup = ({
           justifyContent="flex-end"
           className={styles.allAchievements}
         >
-          {achievementGroup.achievements.length > 1 &&
-            achievementGroup.achievements.map((achievement: any, i: number) => (
-              <Tooltip
-                key={i}
-                positions={['top', 'bottom', 'right', 'left']}
-                content={
-                  <Flex
-                    column
-                    gap="var(--spacing-sm)"
-                    alignItems="center"
-                    justifyContent="center"
+          {achievementGroup.achievements.length > 1 ||
+            (!achievementGroup.isLeveled &&
+              achievementGroup.achievements.map(
+                (achievement: any, i: number) => (
```

Comment:
Shouldn't the leveled and non leveled be listed equally? I though only the logic displaying which is achieved should be changed?